psi4/src/read_options.cc

```diff
@@ -303,13 +303,6 @@ int read_options(const std::string &name, Options &options, bool suppress_printi
         /*- Thole damping factor for multipole fields -*/
         options.add_double("DAMPING_FACTOR_MULTIPOLE", 2.1304);
 
-        /*- Summation scheme for field computations, can be direct or fmm -*/
-        options.add_str_i("SUMMATION_FIELDS", "DIRECT", "DIRECT FMM");
-        /*- Expansion order of the multipoles for FMM -*/
-        options.add_int("TREE_EXPANSION_ORDER", 5);
-        /*- Opening angle theta -*/
-        options.add_double("TREE_THETA", 0.5);
```

Comment:
these are for PE, I think. why remove?

Reply:
Yes, please don't remove those options 😄